week06/6.1.count_specials.py

```diff
@@ -0,0 +1,32 @@
+#================================
+# 6.1 - ספירת מלאי
+#================================
+from check_type_decorator import check_type
+
+
+#--------------------------------------------------------------------------
+# return how many numbers that smaller than n and are divisible by 3 or 7
+#--------------------------------------------------------------------------
```

Comment:
Use docstring instead
https://www.pythonforbeginners.com/comments/when-to-use-comments-vs-docstrings-in-python